.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -13,14 +13,7 @@
 NVIDIA-GH200:
   extends: .common-setup
   tags: [nvidia-gh200]
-  image: nvidia/cuda:12.6.1-devel-ubuntu24.04
-  rules:
-    - if: $CI_PIPELINE_SOURCE == "schedule"
-      variables:
-        CDASH_MODEL: "Nightly"
-    - when: on_success
-      variables:
-        CDASH_MODEL: "Experimental"
+  image: nvcr.io/nvidia/cuda:12.6.1-devel-ubuntu24.04
```

Comment:
I prepended `nvcr.io/` because I think it is better to get the image from NVIDIA's official registry than DockerHub